src/commonMain/kotlin/org/snakeyaml/engine/v2/exceptions/MarkedYamlEngineException.kt

```diff
@@ -20,7 +20,7 @@ package org.snakeyaml.engine.v2.exceptions
  */
 open class MarkedYamlEngineException private constructor(
     // can't override `message` property, so define a new one
-    // https://youtrack.jetbrains.com/issue/KT-43490/
+    // https://youtrack.jetbrains.com/issue/KT-52653/
```

Comment:
Hmm, both tickets are closed 🤔 Do you have some more context on this?

Reply:
Hmm looks like this commit got snuck in, and shouldn't be in this PR. I'll remove it.

But also, yes, it looks like it's fixed in Kotlin 1.9 so when we update we can fix it too.

Reply:
```suggestion
    // https://youtrack.jetbrains.com/issue/KT-43490/
```